registration-processor/post-processor/registration-processor-printing-stage/src/main/resources/bootstrap.properties

```diff
@@ -22,4 +22,9 @@ eventbus.port=5722
 mosip.regproc.eventbus.kafka.commit.type=${mosip.regproc.printing.eventbus.kafka.commit.type}
 mosip.regproc.eventbus.kafka.max.poll.records=${mosip.regproc.printing.eventbus.kafka.max.poll.records}
 mosip.regproc.eventbus.kafka.poll.frequency=${mosip.regproc.printing.eventbus.kafka.poll.frequency}
-mosip.regproc.eventbus.kafka.group.id=${mosip.regproc.printing.eventbus.kafka.group.id}
\ No newline at end of file
+mosip.regproc.eventbus.kafka.group.id=${mosip.regproc.printing.eventbus.kafka.group.id}
+mosip.registration.processor.digitalcard.credentialtype=PDFCard
+mosip.registration.processor.print.credentialtype=euin
+mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
+mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
```

Comment:
Add detailed description about the property with an example.

Reply:
taken care